demo/index.js

```diff
@@ -18,3 +18,23 @@ createFocusGroup({
 	},
 	wrap: true,
 }).activate();
+
+var selectRoot = document.getElementById('select-root');
+var selectNodes = document.querySelectorAll('.select');
+var focusGroup = createFocusGroup({
+  rootNode: selectRoot,
+	members: selectNodes,
+	stringSearch: true,
+	keybindings: {
+		next: [{ keyCode: 40 }, { keyCode: 39 }],
+		prev: [{ keyCode: 38 }, { keyCode: 37 }]
+	},
+	wrap: true,
+}).activate();
+
+focusGroup.on('focus', function (member) {
```

Comment:
Totally agree. Didn't know what else to call it. Freaking naming is hard 😅  I like `move` though 👍  I had `change` in there before, but didn't know if that fit or not.